README.md

```diff
@@ -2,6 +2,9 @@
 
 Provides standard objects and functions for working with dates and times.
 
+**NOTE: The current working version is the [Polyfill](./polyfill). Specification text is being written based on that!**
+***This is NOT suitable for production use as it has several performance issues that are in place for the sake of clarity!!!***
```

Comment:
Can I ask why? I think it'll be good to have a production-ready polyfill at some point.

Reply:
I agree, it would be useful to have an official polyfill developers can use in their applications, which would make the transition from Date to Temporal even easier.

Reply:
It would be useful, but a) proposals generally aren’t recommended for use in production prior to stage 3 anyways, and b) a production-ready polyfill is maximally backwards compatible, which would include not using bigints and symbols etc directly.